xiangqi updated/src/Pages/Board.scss

```diff
@@ -0,0 +1,266 @@
+body {
+  background-color: #f4f4f9;
+  margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-family: 'Arial', sans-serif;
+}
+
+.board-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: flex-start;
+  width: 100%;
+  max-width: 1200px;
+  padding: 20px;
+  box-shadow: 0 0 15px rgba(0, 0, 0, 0.2);
+  border-radius: 10px;
+  background-color: #e8e4e1; 
+  gap: 20px;
+}
+
+.board {
+  width: 80vmin;
+  height: 80vmin;
+  display: grid;
+  grid-template-rows: repeat(10, 1fr);
+  grid-template-columns: repeat(9, 1fr);
+  gap: 2px;
+  border: 4px solid #d4a373;
+  background-color: #f7f1e1;
+  box-shadow: 0 0 15px rgba(0, 0, 0, 0.3);
+  border-radius: 10px;
+}
+
+.move-history {
+  width: 250px;
+  max-height: 80vmin;
+  overflow-y: auto;
+  font-size: 16px;
+  color: #333;
+  background-color: #f4f4f4; 
+  padding: 15px;
+  border-radius: 10px;
+  box-shadow: 0 0 15px rgba(0, 0, 0, 0.3);
+}
+
+.turn-section {
+  width: 250px;
+  background-color: #dcdcdc; 
+  padding: 15px;
+  border-radius: 10px;
+  box-shadow: 0 0 15px rgba(0, 0, 0, 0.3);
+  margin-left: 20px; 
+}
+
+.row {
+  display: contents;
+}
+
+.river {
+  background-color: skyblue !important;
+  border: none !important;
+  transition: none;
+}
+
+.piece {
+  width: 90%; 
+  height: 90%;
+  padding: 10%; 
+  margin: 0; 
+  border-radius: 50%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-size: 80%; 
+  background-position: center;
+  background-repeat: no-repeat; 
+  background-origin: content-box; 
+  background-clip: content-box; 
+  transition: transform 0.3s;
+  box-sizing: border-box;
+}
+
+.cell {
+  position: relative;
+  width: 100%;
+  height: 100%;
+  padding: 5%;
+  border: 1px solid #d4a373;
+  background-color: #e2c7a1;
+  transition: background-color 0.3s, transform 0.3s;
+  border-radius: 5px;
+  box-shadow: 0 0 5px rgba(0, 0, 0, 0.1);
+}
+
+.cell:hover {
+  background-color: #d0a971;
+  transform: scale(1.05);
+}
+
+.piece:hover {
+  transform: scale(1.1);
+}
+
+.cell.valid-move {
+  background-color: rgba(0, 255, 0, 0.3);
+}
+
+.cell.selected {
+  background-color: rgba(255, 255, 0, 0.5);
+  border: 2px solid #00f;
+}
```

Comment:
This should come under `.cell {` and not separately !

Reply:
i'll discard this particular board in my next PR